### PR TITLE
refactor: 💡 remove jquery

### LINKED ISSUE #106: Remove jQuery

### PR DESCRIPTION
**Pull request by iam-robin**
closes #106 

Since jQuery is outdated, all references have been removed from onboarding.
jQuery is only mentioned in one place to make clear that it should no longer be used.

### CODE REVIEW

File: src/security.md [hunk 1]
```diff
@@ -39,11 +39,11 @@ See following types of data and the XSS Game below for real-world examples.
 Incoming data from URL parameters is considered the most critical security risk.
 This data might be easily injected from incoming links even if the website is read-only.
 
-Some very simple and unsafe jQuery snippet example:
+Some very simple and unsafe JS snippet example:
 ```
 // Example-URL: `www.domain.com?search=foo`
 // ... grab parameter directly from URL and paste the search word above the results
-$('.search__searchPhrase').html(urlParams['search']);
+document.getElementsByClassName('search__searchPhrase')[0].html(urlParams['search']);
```
Comment (luke-m) [inline src/security.md:46]:
```suggestion
document.querySelector('search__searchPhrase').innerHTML = urlParams['search'];
```

Why not `querySelector`? And `.html` is a jQuery method (afaik), so use `.innerHTML = ...` instead.

Comment (iam-robin) [reply src/security.md]: Thanks Luke for the Feedback. I've fixed the JS code example and tried to give more context why we're trying to move away from jQuery.

**Comment by luke-m** [comment]
Me gusta. 🚀

File: src/security.md [hunk 2]
```diff
@@ -39,11 +39,11 @@ See following types of data and the XSS Game below for real-world examples.
 Incoming data from URL parameters is considered the most critical security risk.
 This data might be easily injected from incoming links even if the website is read-only.
 
-Some very simple and unsafe jQuery snippet example:
+Some very simple and unsafe JS snippet example:
 ```
 // Example-URL: `www.domain.com?search=foo`
 // ... grab parameter directly from URL and paste the search word above the results
-$('.search__searchPhrase').html(urlParams['search']);
+document.querySelector('.search__searchPhrase')innerHtml = urlParams['search'];
```
Comment (Pustur) [inline src/security.md:46]: There is a missing `.` before `innerHtml`

**Pull request merged**